### PR TITLE
Cps/implementing far field forces

### PR DESCRIPTION
**Pull request by inigolcanalejo**
This PR implements the computation of lift and drag coefficients using the far field fluid magnitudes and applying the integral momentum theorem.

formula_far_field_force

This approach is specially interesting to accurately compute the induced drag in 3D.

### CODE REVIEW

File: applications/CompressiblePotentialFlowApplication/python_scripts/compute_far_field_lift_process.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+        KratosMultiphysics.Logger.PrintInfo('ComputeFarFieldLiftProcess',' Cl = ', self.lift_coefficient)
+        KratosMultiphysics.Logger.PrintInfo('ComputeFarFieldLiftProcess',' Cd = ', self.drag_coefficient)
+
+        self.fluid_model_part.ProcessInfo.SetValue(CPFApp.LIFT_COEFFICIENT_FAR_FIELD, self.lift_coefficient)
+        self.fluid_model_part.ProcessInfo.SetValue(CPFApp.DRAG_COEFFICIENT_FAR_FIELD, self.drag_coefficient)
+
+    def __CalculateWakeTangentAndNormalDirections(self):
+        self.wake_direction = self.fluid_model_part.ProcessInfo.GetValue(CPFApp.FREE_STREAM_VELOCITY)
+        if(self.wake_direction.Size() != 3):
```
Comment (philbucher) [inline applications/CompressiblePotentialFlowApplication/python_scripts/compute_far_field_lift_process.py]: I think this is an array3-variable, hence you don't need to check the size

**Review by marcnunezc — DISMISSED** [review]
Nice! I have no comments regarding the code.

Only one question: at some point we merged the pressure distribution lift and the potential jump lift into the same file. Shall we do the same with the far field lift?

**Comment by inigolcanalejo** [comment]
You are right!

File: applications/CompressiblePotentialFlowApplication/python_scripts/compute_far_field_lift_process.py (hunk 1 above)
Comment (inigolcanalejo) [reply applications/CompressiblePotentialFlowApplication/python_scripts/compute_far_field_lift_process.py]: I removed the check

**Review by marcnunezc — APPROVED** [review]

**Pull request merged**